lib/elixir/pages/Writing Documentation.md

```diff
@@ -74,17 +74,19 @@ Code comments are for developers reading the code. They are useful to mark impro
 
 In other words, documentation is required, code comments are optional.
 
-## Privacy
+## Hiding Internal Modules and Functions
 
-Elixir does not allow developers to document private functions. That's because private functions only exist inside the module that defines them and cannot be accessed externally.
+Libraries often have two levels of public interface: On one hand, they expose an API for external clients. But on the other hand, the modules and functions that conform the library often need to use other modules and functions which are not part of the external public interface.
```

Comment:
Also, I would not put a period before the "But on the other hand...", or alternatively, put a period and then use "On the other hand..." (without "but").